foundations/javascript_basics/problem_solving.md

```diff
@@ -258,3 +251,10 @@ This section contains questions for you to check your understanding of this less
 *   [What is an algorithm?](#algorithm)
 *   [What is pseudocode?](#pseudo)
 *   [What are the advantages of breaking a problem down and solving the smaller problems?](#breaking-problem)
+
+### Additional resources
+
+This section contains helpful links to other content. It isn't required, so consider it supplemental.
+
+*   Read the first chapter in [Think Like a Programmer: An Introduction to Creative Problem Solving](https://nostarch.com/thinklikeaprogrammer) (*not free*). This book's examples are in C++, but you will understand everything since the main idea of the book is to teach programmers to better solve problems. It's an amazing book and worth every penny. It will make you a better programmer.
+*   Watch this [video on repetitive programming techniques](https://ocw.mit.edu/courses/res-tll-004-stem-concept-videos-fall-2013/resources/basic-programming-techniques/).
```

Comment:
```suggestion
- [What is an algorithm?](#algorithm)
- [What is pseudocode?](#pseudo)
- [What are the advantages of breaking a problem down and solving the smaller problems?](#breaking-problem)

### Additional resources

This section contains helpful links to other content. It isn't required, so consider it supplemental.

- Read the first chapter in [Think Like a Programmer: An Introduction to Creative Problem Solving](https://nostarch.com/thinklikeaprogrammer) (*not free*). This book's examples are in C++, but you will understand everything since the main idea of the book is to teach programmers to better solve problems. It's an amazing book and worth every penny. It will make you a better programmer.
- Watch this [video on repetitive programming techniques](https://ocw.mit.edu/courses/res-tll-004-stem-concept-videos-fall-2013/resources/basic-programming-techniques/).
```